packages/storybook-builder-vite/codegen-importfn-script.js

```diff
@@ -16,7 +16,15 @@ const { normalizePath } = require('vite');
  */
 async function toImportFn(stories) {
     const objectEntries = stories
-        .map(file => `  './${normalizePath(path.relative(process.cwd(),file))}': async () => import('/@fs/${file}')`);
+        .map(file => {
```

Comment:
I'm not sure the naive approach of `toImportPath` works here, since I think it fails for files and folders starting with a `.` (such as hidden files/folders like `.storybook`).  

`normalizePath` from vite is a wrapper around `path.posix.normalize` which handles windows paths (https://github.com/vitejs/vite/blob/b40163c794b1c3be3f1715ed8a04d03dc72388b4/packages/vite/src/node/utils.ts#L83`).  I believe that `path.normalize` will never include `./` or `../` at the start, so I'm not sure what the change here is attempting to solve.  @kazuma1989 can you explain why you're checking for relative paths like this?

Reply:
ah hah, that makes sense.  So I guess we'd just need to check for `../`, and only add `./` if it's not there, right?

Reply:
Yep :) This fix is fine. Just wanted to see if there was Storybook code we could copy but seems like there isn't :(

Reply:
Technically the `!normalizedPath.startsWith('./') &&` part of this fix is not necessary, right?

Reply:
Good point. It's not needed :)

Reply:
Thank you so much, @joshwooding and @IanVS!

Yes, @joshwooding gave a perfect explanation.
And the point @IanVS made is also right. Checking `./` is not needed. I added that part just because I didn't know `normalizePath` returns a leading `./` or not.

Fixed in https://github.com/eirslett/storybook-builder-vite/pull/172/commits/8331490662708a9f4cba4da280968790285264b1